manager/manager/utils.py

```diff
@@ -652,28 +653,72 @@ def handle_jira_payload(request, lfa_urls=[]):
 
 
 def get_jira_obs_report(request_data):
-    """Query all issues of the OBS project for a certain day.
-    Then get the total observation time loss from the time_lost param
-    """
+    """Connect to the Rubin Observatory JIRA Cloud REST API to
+    query all issues of the OBS project for a certain obs day.
+
+    For more information on the REST API endpoints refer to:
+    - https://developer.atlassian.com/cloud/jira/platform/rest/v3
+    - https://developer.atlassian.com/cloud/jira/platform/\
+        basic-auth-for-rest-apis/
+
+    Parameters
+    ----------
+    request_data : `dict`
+        The request data
+
+    Notes
+    -----
+    Note that the JIRA REST API query is based on the user timezone so
```

Comment:
Just an esthetic comment: You start the sentence with `Note that the JIRA REST API...` while you write this in de `Notes` section. Just write `The JIRA REST API...` and it's fine. Up to you if you want to change this or not. It is perfectly fine as it is.

Reply:
You are right, its kind of redundant jaja, I'll do the change :+1:. Thank you!